flaskr/templates/users/profile.html

```diff
@@ -1,13 +1,25 @@
 {% extends 'base.html' %}
 
 {% block header %}
+window.onload()
```

Comment:
I think this is automatically invoked so the call here might be unnecessary (we should defer to the system to decide when it is loaded)